base/utils/fs.go

```
@@ -38,7 +41,13 @@
 		if err != nil {
 			return fmt.Errorf("could not create dir %s: %w", path, err)
 		}
-		return os.Chmod(path, perm)
+		if isWindows {
+			// Ignore windows permission error. For none admin users it will always fail.
+			acl.Chmod(path, perm)
```

golangci-lint (1.62.2)

Error return value is not checked

(errcheck)